### PR TITLE
fix: show progress of existing analytics and resource tables tasks

### PR DESCRIPTION
**Pull request by mediremi**
Closes https://jira.dhis2.org/browse/DHIS2-11560

Also fixes an issue where the parameters of the analytics task (e.g. `Skip generation of resource tables`) were set to true even if the user had not checked the associated checkbox.

### CODE REVIEW

File: src/get-active-task-id-from-summary.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export const getActiveTaskIdFromSummary = taskSummaryResponse => {
```
Comment (mediremi) [inline src/get-active-task-id-from-summary.js:1]: This was taken from https://github.com/dhis2/data-administration-app/blob/patch/2.36.3/src/pages/analytics/Analytics.js#L229-L253

File: src/components/NotificationsTable/NotificationsTable.js [hunk 2]
```diff
@@ -35,7 +31,7 @@ const NotificationsTable = ({ notifications, animateIncomplete }) => {
                     {notificationIconInfo.icon}
                 </FontIcon>
             )
-        } else if (animateIncomplete && !notification.completed && !done) {
```
Comment (mediremi) [inline src/components/NotificationsTable/NotificationsTable.js:38]: Turns out this is not needed anymore

File: src/pages/analytics/Analytics.js [hunk 3]
```diff
@@ -30,26 +31,39 @@ const startAnalyticsTablesGenerationMutation = {
...
     })
-    const poll = useAnalyticsPoll()
     const notifications = poll.data ? getUpdatedNotifications(poll.data) : []
-    const isPolling =
-        data && (!poll.data || !notifications.some(n => n.completed))
 
     const handleStartAnalyticsTablesGeneration = () => {
         const params = {}
-        for (const [key, checked] of Object.entries(checkboxes)) {
+        for (const [key, { checked }] of Object.entries(checkboxes)) {
```
Comment (mediremi) [inline src/pages/analytics/Analytics.js:66]: This is where the cause of the bug regarding the analytics parameter values always being `true`

**Review by varl — APPROVED** [review]
Cool, works nicely.

**Pull request merged**